libcontainer/rootfs_linux.go

```diff
@@ -215,6 +215,10 @@ func prepareRootfs(pipe *syncSocket, iConfig *initConfig) (err error) {
 		return fmt.Errorf("error jailing process inside rootfs: %w", err)
 	}
 
+	if err := adjustRootMountPropagation(uintptr(config.RootPropagation)); err != nil {
+		return err
+	}
+
```

Comment:
Yeah, I think that we can use config.RootPropagation directly, and not use a separate function.

Something like
```go
// Apply root mount propagation flags.
if config.RootPropagation != 0 {
 	if err := mount("", "/", "", uintptr(flag), ""); err != nil { 
 		return fmt.Errorf("unable to apply root propagation flags: %w", err )
 	} 
}
```

Reply:
@kolyshkin @lifubang 
Thank you for the review. I've fixed.